rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -81,11 +86,28 @@ def __init__(
         self.stability_pool_contract = self.ethereum.contracts.contract(string_to_evm_address('0x66017D22b0f8556afDd19FC67041899Eb65a21bb'))  # noqa: E501
         self.staking_contract = self.ethereum.contracts.contract(string_to_evm_address('0x4f9Fbb3f1E99B56e0Fe2892e623Ed36A76Fc605d'))  # noqa: E501
 
+    def _calculate_total_collateral_ratio(self, eth_price: Price) -> FVal:
+        """Query Liquity smart contract for Total Collateral Ratio (TCR).
+        If the TCR of the system falls below 150% the system enters recovery Mode"""
+        try:
+            total_collateral_ratio = self.trove_manager_contract.call(
+                node_inquirer=self.ethereum,
+                method_name='getTCR',
+                arguments=[FVal(eth_price * 10**18).to_int(True)],
+            )
+
+        except RemoteError as e:
+            log.error(f'Failed to query liquity contract for protocol collateral ratio: {e}')
+            total_collateral_ratio = None
```

Comment:
This is a bug. Will be multiplied by 100 right below and BOOM an exception will be raised.

You should make the it return `None` here and handle it properly where the function is called